_data/people/fkw888.yml

```diff
@@ -6,6 +6,12 @@ institution: University of California, San Diego
 website: https://www-physics.ucsd.edu/fac_staff/fac_profile/faculty_description.php?person_id=494
 photo: /assets/images/team/Frank-Wuerthwein.jpg
 presentations:
+  - Title: "Advancing Open Science through distributed High Throughput Computing"
+    date: 12 Apr 2019
+    url: https://drive.google.com/file/d/1M8w5ndUURkrsDDomhPFpGco4e0efHCVq/view?usp=sharing
+    meeting: "UCSD CI Engineering Brwon Bag"
+    meetingurl: https://drive.google.com/drive/folders/1jrjmOGvuVA6RsxvkNFlR1dLXc_t8r7Od
```

Comment:
This URL is not obviously about the meeting. I would omit this line, unless you can find a human-readable web page that describes or is about the brown bag series or the group that runs it.